include/shisen_cpp/viewer/consumer/image_consumer.hpp

```diff
@@ -18,43 +18,36 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-#ifndef SHISEN_CPP__CONSUMER__IMAGE_CONSUMER_HPP_
-#define SHISEN_CPP__CONSUMER__IMAGE_CONSUMER_HPP_
+#ifndef SHISEN_CPP__VIEWER__CONSUMER__IMAGE_CONSUMER_HPP_
+#define SHISEN_CPP__VIEWER__CONSUMER__IMAGE_CONSUMER_HPP_
 
-#include <rclcpp/rclcpp.hpp>
+#include <shisen_interfaces/msg/image.hpp>
 
+#include <opencv2/core.hpp>
+#include <opencv2/highgui.hpp>
 #include <memory>
 #include <string>
 
-#include "../node.hpp"
+#include "shisen_cpp/utility.hpp"
 
-namespace shisen_cpp
+
+namespace shisen_cpp::viewer
 {
+using Image = shisen_interfaces::msg::Image;
 
-class ImageConsumer : public CameraNode
+class ImageConsumer
 {
 public:
-  using ImageCallback = std::function<void (const Image &)>;
-
-  struct Options : public virtual CameraNode::Options
-  {
-  };
-
-  explicit ImageConsumer(
-    rclcpp::Node::SharedPtr node, const Options & options = Options());
-
-  ~ImageConsumer();
-
   virtual void on_image_changed(const Image & image);
```

Comment:
Why did you add `virtual` specifier here?

Reply:
I forgot to delete that after refactoring from the previous